copycat/runtime/src/main/java/org/apache/kafka/copycat/runtime/WorkerSinkTask.java

```diff
@@ -187,6 +197,7 @@ public void commitOffsets(boolean sync, final int seqno) {
                 log.debug("{} Rewinding topic partition {} to offset {}", id, entry.getKey(), entry.getValue().offset());
                 consumer.seek(entry.getKey(), entry.getValue().offset());
             }
+            currentOffsets = new HashMap<>(lastCommittedOffsets);
             workThread.onCommitCompleted(t, seqno);
             return;
         }
```

Comment:
I don't have a good place to put this... on line 208, if commitSync was succesfull, we still need to call workerThread.onCommit().